lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -525,6 +525,22 @@ def initialize(target, unresolved_alias, encoding)
       end
     end
 
+    # Represents a global variable e.g.: $DEBUG
+    class GlobalVariable < Entry
+      sig do
+        params(
+          name: String,
+          file_path: String,
+          location: RubyIndexer::Location,
+          comments: T.nilable(String),
+          encoding: Encoding,
+        ).void
+      end
+      def initialize(name, file_path, location, comments, encoding)
+        super(name, file_path, location, comments, encoding)
```

Comment:
```suggestion
        super
```
(since the arguments are identical)